Extension/src/LanguageServer/client.ts

```diff
@@ -689,6 +689,7 @@ export interface Client {
     RootPath: string;
     RootRealPath: string;
     RootUri?: vscode.Uri;
+    RootFolder?: vscode.WorkspaceFolder;
```

Comment:
If you add this to the interface, I think you need to add it to the NullClient as well.

Reply:
I believe this is implicit.  The `?` indicates the value does not need to be defined, and it's correct for `NullClient` to not define it.  (It doesn't have a value to return).